plutus-tx/test/Spec.hs

```diff
@@ -118,3 +121,49 @@ genData =
         , List <$> constructorArgList
         , Map <$> kvMapList
         ]
+
+ratioTests :: TestTree
+ratioTests = testGroup "Ratio"
+  [ testProperty "reciprocal ordering 1" reciprocalOrdering1
+  , testProperty "reciprocal ordering 2" reciprocalOrdering2
+  , testProperty "reciprocal ordering 3" reciprocalOrdering3
+  ]
+
+genPositiveRational :: Monad m => PropertyT m Rational
+genPositiveRational = do
+  a <- forAll . Gen.integral $ Range.linear 1 100000
+  b <- forAll . Gen.integral $ Range.linear 1 100000
+  return (a % b)
+
+genNegativeRational :: Monad m => PropertyT m Rational
+genNegativeRational = negate <$> genPositiveRational
+
+-- If x and y are positive rational numbers and x < y then 1/y < 1/x
+reciprocalOrdering1 :: Property
+reciprocalOrdering1 = property $ do
+  x <- genPositiveRational
+  y <- genPositiveRational
+  if x < y
+  then assert (recip y < recip x)
+  else if y < x
+  then assert (recip x < recip y)
+  else return ()
+
+-- If x and y are negative rational numbers and x < y then 1/y < 1/x
+reciprocalOrdering2 :: Property
+reciprocalOrdering2 = property $ do
+  x <- genNegativeRational
+  y <- genNegativeRational
+  if x < y
+  then assert (recip y < recip x)
+  else if y < x
+  then assert (recip x < recip y)
+  else return ()
+
+-- If x is a negative rational number and y is a positive rational number and
+-- x < y then 1/x < 1/y
+reciprocalOrdering3 :: Property
+reciprocalOrdering3 = property $ do
+  x <- genNegativeRational
+  y <- genPositiveRational
+  when (x < y) $ assert (recip x < recip y)
```

Comment:
`x` is negative and `y` is positive and neither are 0, so `x<y` always holds

Reply:
Oh yes, excellent point. I'll push a change.